pkg/kubectl/cmd/taint.go

```diff
@@ -248,12 +254,12 @@ func (o *TaintOptions) Complete(f cmdutil.Factory, out io.Writer, cmd *cobra.Com
 	o.builder = resource.NewBuilder(mapper, f.CategoryExpander(), typer, resource.ClientMapperFunc(f.ClientForMapping), f.Decoder(true)).
 		ContinueOnError().
 		NamespaceParam(namespace).DefaultNamespace()
+	if o.selector != "" {
+		o.builder = o.builder.SelectorParam(o.selector).ResourceTypes("node")
+	}
 	if o.all {
 		o.builder = o.builder.SelectAllParam(o.all).ResourceTypes("node")
 	} else {
-		if len(o.resources) < 2 {
```

Comment:
@aveshagarwal  So, in the below command, I have a resource `node` without selector and all options. Its throwing a valid error. 
`cluster/kubectl.sh taint node 127.0.0.1 `
`error: at least one taint update is required`